packages/rum-core/src/common/constants.js

```diff
@@ -66,13 +66,15 @@ const MAX_SPAN_DURATION = 5 * 60 * 1000
 const PAGE_LOAD = 'page-load'
 const ROUTE_CHANGE = 'route-change'
 const TYPE_CUSTOM = 'custom'
+const USER_INTERACTION = 'user-interaction'
 const HTTP_REQUEST_TYPE = 'http-request'
 const TEMPORARY_TYPE = 'temporary'
 const NAME_UNKNOWN = 'Unknown'
 
 const TRANSACTION_TYPE_ORDER = [
   PAGE_LOAD,
   ROUTE_CHANGE,
+  USER_INTERACTION,
```

Comment:
Clicks would trigger a route change, not the other way around. I am not 100% sure if this order would work out for us. WDYT?

Reply:
This order means that a `user-interaction` transaction can be redefined as `route-change` but not the other way around. That seems to me the way it should be, Why do you think this doesn't work?